packages/components/src/components/loaders/ProgressBar/ProgressBar.tsx

```diff
@@ -1,35 +1,58 @@
-import styled from 'styled-components';
+import styled, { useTheme } from 'styled-components';
 
-const Wrapper = styled.div`
-    background: ${({ theme }) => theme.backgroundNeutralSubdued};
+import { CSSColor } from '@trezor/theme';
+
+const Wrapper = styled.div<{ $color: CSSColor }>`
+    background: ${({ $color }) => $color};
     width: 100%;
     overflow: hidden;
 `;
 
 type ValueProps = {
     $max: number;
     $value: number;
+    $color: CSSColor;
 };
 
 const Value = styled.div<ValueProps>`
-    background: ${({ theme }) => theme.iconPrimaryDefault};
+    background: ${({ $color }) => $color};
     height: 5px;
     max-width: 100%;
-    width: 1%;
-    transform: ${({ $max, $value }) => `scaleX(${(100 / $max) * $value})`};
-    transform-origin: left;
-    transition: transform 0.5s;
+    width: ${({ $max, $value }) => `calc((100% / ${$max}) * ${$value})`};
```

Comment:
I've changed this because it's better to use transform than to animate width:) It should be the same, do you think it bug is caused by this?

Reply:
Yes, I'm pretty sure it was, now it's working fine